arelle/plugin/validate/ESEF_2022/Dimensions.py

```diff
@@ -0,0 +1,178 @@
+'''
+Created on June 6, 2018
+Version 2022 created on September 19, 2022
+
+Filer Guidelines: ESMA_ESEF Manula 2019.pdf
```

Comment:
nit: typo in `Manual`

Reply:
I have changed the whole name and added the same reference as in arelle/plugin/validate/ESEF_2022/__init__.py

The changes will be pushed soon